website/docs/r/logpush_job.html.markdown

```diff
@@ -26,12 +27,17 @@ resource "cloudflare_logpush_job" "example_job" {
 
 The following arguments are supported:
 
-* `desitination_conf` - (Required) Uniquely identifies a resource (such as an s3 bucket) where data will be pushed. Additional configuration parameters supported by the destination may be included.
-* `ownership_challenge` - (Required) Ownership challenge token to prove destination ownership. See [https://developers.cloudflare.com/logs/tutorials/tutorial-logpush-curl/](https://developers.cloudflare.com/logs/tutorials/tutorial-logpush-curl/)
+
+* `name` - (Required) The name of the logpush job to create. Must match the following regular expression (ie. does not support `_`) : "^[a-zA-Z0-9\-\.]*$"
+* `zone_id` - (Required) The zone ID where the logpush job should be created.
+* `desitination_conf` - (Required) Uniquely identifies a resource (such as an s3 bucket) where data will be pushed. Additional configuration parameters supported by the destination may be included. See [https://developers.cloudflare.com/logs/logpush/logpush-configuration-api/understanding-logpush-api/#destination](https://developers.cloudflare.com/logs/logpush/logpush-configuration-api/understanding-logpush-api/#destination)
+* `ownership_challenge` - (Required) Ownership challenge token to prove destination ownership. See [https://developers.cloudflare.com/logs/logpush/logpush-configuration-api/understanding-logpush-api/#usage](https://developers.cloudflare.com/logs/logpush/logpush-configuration-api/understanding-logpush-api/#usage)
+* `logpull_options` - (Optional) the logpull_options job parameter. See [https://developers.cloudflare.com/logs/logpush/logpush-configuration-api/understanding-logpush-api/#options](https://developers.cloudflare.com/logs/logpush/logpush-configuration-api/understanding-logpush-api/#options)
+* `enable` - (Optional) Whether to enable to job to create or not.
 
 ## Import
 
-Logpush jobs can be imported using a composite ID formed of zone ID and logpush job ID, e.g.
+Once logpush job supports import, it can imported using a composite ID formed of zone ID and logpush job ID, e.g.
```

Comment:
If this isn't something we support at the moment, let's just get rid of the import section entirely.